src/components/version-context-switcher/index.tsx

```diff
@@ -92,9 +61,7 @@ const VersionContextSwitcher = ({
 					</option>
 				))}
 			</select>
-			<span className={s.trailingIcon}>
-				<IconCaret16 />
-			</span>
+			<IconChevronDown16 className={s.trailingIcon} tab-index="-1" />
```

Comment:
While I do appreciate the tab-index, i do not believe it is necessary. From my, so far limited and new, understanding of our codebase this chevron is this icon:

<img width="1312" alt="Screenshot showcasing what the author believes to be the chevron icon" src="https://github.com/hashicorp/dev-portal/assets/1957315/bcaed4a8-4411-4921-9f31-3cf5953aae3f">

Correct? Which if so then this Icon renders as an SVG, and SVGs are not natively marked as "interactive" so would not be in the "focus tree". So no tab-index would be required.